HardwareDefinitions/avnet_mt3620_sk/sample_appliance.json

```diff
@@ -25,19 +25,19 @@
-        {"Name": "SAMPLE_UART_LOOPBACK", "Type": "Uart", "Mapping": "AVNET_MT3620_SK_ISU0_UART", "Comment": "MT3620 SK: Connect SOCKET1 RX (RX) to SOCKET1 TX (TX)."},
-        {"Name": "SAMPLE_LSM6DS3_I2C", "Type": "I2cMaster", "Mapping": "AVNET_MT3620_SK_ISU1_I2C", "Comment": "MT3620 SK: Connect external LSM6DS3 to I2C using SOCKET1, pin MISO (SDA) and pin MOSI (SCL)."},
-        {"Name": "SAMPLE_LSM6DS3_SPI", "Type": "SpiMaster", "Mapping": "AVNET_MT3620_SK_ISU1_SPI", "Comment": "MT3620 SK: Connect external LSM6DS3 to SPI using SOCKET1, pin MISO (MISO), pin SCK (SCLK), pin MOSI (MOSI) and SOCKET2 pin CS (CSB)."},
+        {"Name": "SAMPLE_UART_LOOPBACK", "Type": "Uart", "Mapping": "AVNET_MT3620_SK_ISU0_UART", "Comment": "MT3620 SK: Connect CLICK1 RX (RX) to CLICK1 TX (TX)."},
+        {"Name": "SAMPLE_LSM6DS3_I2C", "Type": "I2cMaster", "Mapping": "AVNET_MT3620_SK_ISU2_I2C", "Comment": "MT3620 SK: Connect external LSM6DS3 to I2C using CLICK1, pin MISO (SDA) and pin MOSI (SCL)."},
+        {"Name": "SAMPLE_LSM6DS3_SPI", "Type": "SpiMaster", "Mapping": "AVNET_MT3620_SK_ISU1_SPI", "Comment": "MT3620 SK: Connect external LSM6DS3 to SPI using CLICK1, pin MISO (MISO), pin SCK (SCLK), pin MOSI (MOSI) and CLICK2 pin CS (CSB)."},
         {"Name": "SAMPLE_LSM6DS3_SPI_CS", "Type": "int", "Mapping": "MT3620_SPI_CS_B", "Comment": "MT3620 SPI Chip Select (CS) value \"B\". This is not a peripheral identifier, and so has no meaning in an app manifest."},
-        {"Name": "SAMPLE_NRF52_RESET", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO28", "Comment": "MT3620 SK: Connect external reset signal using SOCKET1: RX."},
-        {"Name": "SAMPLE_NRF52_DFU", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO26", "Comment": "MT3620 SK: Connect external dfu signal using SOCKET1: TX."},
-        {"Name": "SAMPLE_NRF52_UART", "Type": "Uart", "Mapping": "AVNET_MT3620_SK_ISU1_UART", "Comment": "MT3620 SK: Connect external NRF52 UART using SOCKET1: \"MISO\" (RX), \"SCK\" (TX), \"CS\" (CTS), and \"MOSI\" (RTS)."},
-        {"Name": "SAMPLE_DEVICE_STATUS_LED", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO0", "Comment": "MT3620 SK: Connect external red LED using SOCKET1, pin PWM."},
-        {"Name": "SAMPLE_PENDING_UPDATE_LED", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO2", "Comment": "MT3620 SK: Connect external blue LED using SOCKET1, pin PWM2."},
+        {"Name": "SAMPLE_NRF52_RESET", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO17", "Comment": "MT3620 SK: Connect external NRF52 RESET signal using the PMOD connector: Pin-8."},
+        {"Name": "SAMPLE_NRF52_DFU", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_GPIO2", "Comment": "MT3620 SK: Connect external NRF52 DFU signal using the PMOD connector: Pin-7"},
+        {"Name": "SAMPLE_NRF52_UART", "Type": "Uart", "Mapping": "AVNET_MT3620_SK_ISU0_UART", "Comment": "MT3620 SK: Connect external NRF52 UART using the PMOD connector): (RX Pin-3), (TX Pin-2), (CTS Pin-1), and (RTS Pin-4)."},
```

Comment:
Why were these changes necessary? We should talk about testing the changes. We will be in contact on a private thread.

Reply:
HI Andrea,

Thanks for digging in to these changes and asking questions.  

There are two logical changes here for the original Starter Kit files, first SAMPLE_LSM6DS3_I2C was changed from ISU1 I2C to ISU2 I2C because ISU I2C signals are not present on either click sockets; so that was never correct.  ISU1 I2C is present on the click sockets so we define that interface for correctness.

The second set of changes for the NRF external update example originally defined the IUS1 UART on the click socket.  While the ISU1 UART TX and RX signals are available on the click sockets, the ISU UART RTS and CTS signals are not.  The only place UART RTS and CTS signals are available on the Starter Kit are on the PMOD connector site which uses ISU0.

I did test the NRF external update changes on my starter kit using the proposed changes and the PMOD signals.  I don't have the LSM6DS3 hardware to test the ISU change, so I did not test that change with the example software.  

Note that the NRF external update instructions include a wiring diagram for the Seeed Development board, there are not any diagrams for wiring the Avnet Starter Kit, so this change does not drive any documentation changes.  Once these changes are accepted and we release the Starter Kit Revision 2, I'll update and submit a pull request to update the documentation for the NRF external update example and add wiring diagrams for both kits.

You can review these interfaces in the Starter Kit Rev1 User Guide here:  https://www.avnet.com/opasdata/d120001/medias/docus/197/Azure%20Sphere%20Starter%20Kit%20User%20Guide%20(v1.5).pdf

Brian

Reply:
A colleague just corrected me on the SAMPLE_LSM6DS3_I2C change I made for the Starter Kit Rev1.  I was incorrect in my statement and should have never changed that definition.  I've made the correction in the branch.